functions/response_streaming/Gemfile.lock

```diff
@@ -0,0 +1,93 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    cloud_events (0.7.0)
+    concurrent-ruby (1.2.2)
+    declarative (0.0.20)
+    faraday (2.7.4)
+      faraday-net_http (>= 2.0, < 3.1)
+      ruby2_keywords (>= 0.0.4)
+    faraday-net_http (3.0.2)
+    functions_framework (0.11.0)
+      cloud_events (>= 0.5.1, < 2.a)
+      puma (>= 4.3.0, < 6.a)
+      rack (~> 2.1)
+    google-apis-bigquery_v2 (0.48.0)
+      google-apis-core (>= 0.11.0, < 2.a)
+    google-apis-core (0.11.0)
+      addressable (~> 2.5, >= 2.5.1)
+      googleauth (>= 0.16.2, < 2.a)
+      httpclient (>= 2.8.1, < 3.a)
+      mini_mime (~> 1.0)
+      representable (~> 3.0)
+      retriable (>= 2.0, < 4.a)
+      rexml
+      webrick
+    google-cloud-bigquery (1.42.0)
+      concurrent-ruby (~> 1.0)
+      google-apis-bigquery_v2 (~> 0.1)
+      google-cloud-core (~> 1.6)
+      googleauth (>= 0.16.2, < 2.a)
+      mini_mime (~> 1.0)
+    google-cloud-core (1.6.0)
+      google-cloud-env (~> 1.0)
+      google-cloud-errors (~> 1.0)
+    google-cloud-env (1.6.0)
+      faraday (>= 0.17.3, < 3.0)
+    google-cloud-errors (1.3.1)
+    googleauth (1.3.0)
+      faraday (>= 0.17.3, < 3.a)
+      jwt (>= 1.4, < 3.0)
+      memoist (~> 0.16)
+      multi_json (~> 1.11)
+      os (>= 0.9, < 2.0)
+      signet (>= 0.16, < 2.a)
+    httpclient (2.8.3)
+    jwt (2.7.0)
+    memoist (0.16.2)
+    mini_mime (1.1.2)
+    multi_json (1.15.0)
+    mustermann (3.0.0)
+      ruby2_keywords (~> 0.0.1)
+    nio4r (2.5.8)
+    os (1.1.4)
+    public_suffix (5.0.1)
+    puma (5.6.5)
+      nio4r (~> 2.0)
+    rack (2.2.6.4)
+    rack-protection (3.0.5)
+      rack
+    representable (3.2.0)
+      declarative (< 0.1.0)
+      trailblazer-option (>= 0.1.1, < 0.2.0)
+      uber (< 0.2.0)
+    retriable (3.1.2)
+    rexml (3.2.5)
+    ruby2_keywords (0.0.5)
+    signet (0.17.0)
+      addressable (~> 2.8)
+      faraday (>= 0.17.5, < 3.a)
+      jwt (>= 1.5, < 3.0)
+      multi_json (~> 1.10)
+    sinatra (3.0.5)
+      mustermann (~> 3.0)
+      rack (~> 2.2, >= 2.2.4)
+      rack-protection (= 3.0.5)
+      tilt (~> 2.0)
+    tilt (2.1.0)
+    trailblazer-option (0.1.2)
+    uber (0.1.0)
+    webrick (1.8.1)
+
+PLATFORMS
+  arm64-darwin-22
```

Comment:
This will cause problems in CI, as CI runs on linux. Please replatform to `ruby` and remove darwin. You may be able to just replace this line textually, but if that doesn't work, use the `bundle lock --add-platform` and `bundle lock --remove-platform` commands (see https://bundler.io/v2.4/man/bundle-lock.1.html).